README.md

```diff
@@ -0,0 +1,73 @@
+# Pydarn 
+[![Open Source Love](https://badges.frapsoft.com/os/mit/mit.svg?v=102)](https://github.com/ellerbrock/open-source-badge/) [![Python 3.6](https://img.shields.io/badge/python-3.6-blue.svg)](https://www.python.org/downloads/release/python-360/) [![GitHub version](https://badge.fury.io/gh/boennemann%2Fbadges.svg)](http://badge.fury.io/gh/boennemann%2Fbadges)
+
+SuperDARN data visualization library. 
+
+## Getting Started 
+
+The following instructions will allow you to install and give some examples on how to use pydarn. 
+
+### Prerequisites
+
+**python 3.6+**
+
+| Ubuntu      | OpenSuse       |
+| ----------- | -------------- |
+| libyaml-dev | python3-PyYAML |
+
+You can check your python version with  
+`$ python --version` or 
+`$ python3 --version`
+### Installing 
+
+1. Clone git repository:   
+   `git clone https://github.com/SuperDARN/pydarn.git`
+
+2. Installing pydarn  
+    1. **Recommended**: Installing a [virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtualenv/), this option allows the library to install needed version of libraries without affecting system libraries.  
+        * First install the environment:  
+      `$ python3 -m pip install --user virtualenv`  
+      `$ python3 -m virtualenv <environment name>`  
+		  `$ source <environment name>/bin/activate`
+        * Navigate to where you cloned pydarn:  
+		  `$ python3 setup.py install`
+    2. Install in the system (root privileges required):  
+		   `$ sudo python3 setup.py install`
+
+### Example 
+
+#### Reading DMAP file 
+DMAP files are a SuperDARN 
```

Comment:
Previously suggest by @ksterne. 

> It would be great if there was a specification published somewhere, so maybe you and Keith could write something up and then it could be referenced here?

I am not sure what you are trying to get at here. What do you mean by "specification published"?  Write what up? 

I think, for now, I was just trying to give an example of what the current state of pydarn could do. Later on, there will be more documentation with better user case examples. However, I don't want to get too far ahead of myself and just keep it simple for now.

Reply:
You and Keith are basically the only 2 people on the planet that know DMAP now. This comment is just a suggestion about something that could potentially be a good thing to do in the future.

I'm not a wizard at github reviews yet, so I didn't understand what I was doing in requiring things vs. leaving comments.

Reply:
I see, sorry, jumped the gun on my part. 

Naturally, I take comments in reviews as something you have to do or discuss on doing especailly when you "request changes to be made." Maybe say this is a suggestion, or perhaps something to consider later is blah, for better clarity?